src/server/lib/callback-handler.js

```diff
@@ -206,19 +206,21 @@ export default async function callbackHandler(
         // we do already have an account with the same email address as the one in the
         // OAuth profile the user has just tried to sign in with.
         //
-        // We don't want to have two accounts with the same email address, and we don't
-        // want to link them in case it's not safe to do so, so instead we prompt the user
-        // to sign in via email to verify their identity and then link the accounts.
-        throw new AccountNotLinkedError()
+        // It's not safe to link email with a new profile, but explicilty allowed
```

Comment:
The entire login flow lives in this particular function in NextAuth

Reply:
The callback function lives here: https://github.com/GitStartHQ/next-auth/pull/1/files#diff-75dae873d6c8175a05288e6d0d0a0cda083b213e8092498442dd458ba290b1a3R21